src/Microsoft.OData.Edm/Validation/InterfaceValidator.cs

```diff
@@ -314,6 +314,22 @@ private IEnumerable<EdmError> ValidateStructure(object item)
                 }
             }
 
+            // Out-of-line annotations are found through the model visitor, but inline annotations
+            // are not found through normal traversal, so add in-line annotations here.
+            IEdmVocabularyAnnotatable annotatable = item as IEdmVocabularyAnnotatable;
+            if (this.model != null && annotatable != null)
+            {
+                foreach (IEdmVocabularyAnnotation annotation in annotatable.VocabularyAnnotations(this.model))
+                {
+                    // Inline annotations have a null target.
+                    if (annotation.Target == null)
+                    {
+                        // Target must be set for validation, so create a new EdmVocabularyAnnotation with the target set
+                        followupErrors.AddRange(this.ValidateStructure(new EdmVocabularyAnnotation(annotatable, annotation.Term, annotation.Qualifier, annotation.Value)));
+                    }
+                }
+            }

```

Comment:
That seems not related to the nav prop binding

Reply:
Right; I ran into this while working through some related issues and just fixed it here rather than opening a separate PR.

---
In reply to: [174974871](https://github.com/OData/odata.net/pull/1109#discussion_r174974871) [](ancestors = 174974871)